ts/utils/staking_utils.ts

```diff
@@ -53,6 +55,19 @@ export const stakingUtils = {
             pool: pools.find(pool => pool.poolId === poolId),
             zrxAmount: formatZrx(stakingDecisions[poolId], { removeComma: true }).roundedValue,
         }), []);
-        return recs;
+
+        // Sort desc
+        const orderedRecs = [...recs.sort((recA, recB) => recB.zrxAmount - recA.zrxAmount)];
```

Comment:
NIT: `array.sort` sorts the array in-place so this would mutate `recs` as well. If you want to keep it immutable you need to do something like `[].concat(recs).sort(...)` or  use`_.sortBy`.

Reply:
Thanks, good catch, will fix